manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MatchTest.scala

```diff
@@ -19,347 +19,305 @@
+          p("Return any nodes connected with the +Person+ Oliver that are labeled +Movie+.")
+          resultTable()
+        }
+      }
+    }
+    section("Relationship basics") {
+      section("Outgoing relationships") {
+        p("When the direction of a relationship is interesting, it is shown by using `-->` or `<--`, like this:")
+        query("MATCH (:Person { name:'Oliver Stone' })-->(movie) RETURN movie.title", assertWallStreetIsReturned) {
+          p("Return any nodes connected with the +Person+ Oliver that are labeled +Movie+.")
+          resultTable()
+        }
+      }
+      section("Directed relationships and identifier") {
+        p("If an identifier is needed, either for filtering on properties of the relationship, or to return the relationship, this is how you introduce the identifier.")
+        query("MATCH (:Person { name:'Oliver Stone' })-[r]->(movie) RETURN type(r)", assertRelationshipIsDirected) {
+          p("Returns all outgoing relationships from Oliver.")
+          resultTable()
+        }
+      }
+      section("Match by relationship type") {
+        p("When you know the relationship type you want to match on, you can specify it by using a colon together with the relationship type.")
+        query("MATCH (wallstreet:Movie { title:'Wall Street' })<-[:ACTED_IN]-(actor) RETURN actor.name", assertAllActorsOfWallStreetAreFound) {
+          p("Returns all outgoing relationships from Oliver.")
+          resultTable()
+        }
+      }
+      section("Match by multiple relationship types") {
+        p("To match on one of multiple types, you can specify this by chaining them together with the pipe symbol `|`.")
+        query("MATCH (wallstreet { title:'Wall Street' })<-[:ACTED_IN|:DIRECTED]-(person) RETURN person.name", assertEveryoneConnectedToWallStreetIsFound) {
+          p("Returns nodes with a +ACTED_IN+ or +DIRECTED+ relationship to Wall Street.")
+          resultTable()
+        }
+      }
+      section("Match by relationship type and use an identifier") {
+        p("If you both want to introduce an identifier to hold the relationship, and specify the relationship type you want, just add them both, like this:")
+        query("MATCH (wallstreet { title:'Wall Street' })<-[r:ACTED_IN]-(actor) RETURN r", assertRelationshipsToWallStreetAreReturned) {
+          p("Returns nodes that +ACTED_IN+ Wall Street.")
+          resultTable()
+        }
+      }
+    }
+    section("Relationships in depth") {
+      note {
+        p("Inside a single pattern, relationships will only be matched once. You can read more about this in <<cypherdoc-uniqueness>>")
+      }
+      section("Relationship types with uncommon characters") {
+        p("Sometime your database will have types with non-letter characters, or with spaces in them. Use +`+ (backtick) to quote these.")
+        query("MATCH (n { name:'Rob Reiner' })-[r:`TYPE THAT HAS SPACE IN IT`]->() RETURN count(*)", assertTypeWithRelsExist) {
```

Comment:
Can we avoid hitting keywords (has, in) in the reltype? The syntax highlighter isn't very smart.